src/react-app-env.d.ts

```diff
@@ -5,6 +5,8 @@ declare module '@metamask/jazzicon' {
 }
 
 interface Window {
+  GIT_COMMIT_HASH?: string
```

Comment:
Isn't this always defined? If we put it in `index.tsx`, at the top, where polyfills are, I think this would be a safe assumption.

Maybe we could have e `window.ts` top-level file next to `polyfills` where we have all the window touching things + export the type.

Reply:
the env variable is typed as `string | undefined`, so I made it optional to match that.
I think this is the idiomatic file to extend Window in, from a CRA perspective.